### ISSUE TITLE
Nail bed defect

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10708]
**Preferred term label:**
Nail bed defect

**Synonyms**

**Definition (free text, please give PubMed ID)**
Laceration through the nail bed, either with or without fingernail in place

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Seymour fracture
Subungual hematoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Since the word "defect" is not very specific, I would like to use the label "Nail bed laceration" is that is what is meant here?

**Comment by MickeySegal** [comment]
That is fine.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001249
- label: Nail bed laceration
- parents: [Abnormal nail morphology (HP:0001597)](https://hpo.jax.org/app/browse/term/HP:0001597)
- pmid: PMID:34633779
- def: A cut or injury to the soft tissue under the fingernail or toenail.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10708
- synonyms:
